### PR TITLE
add an additional Modal size

### PR DESCRIPTION
**Pull request by szaimen**
Follow up to #2126

| Size | Desktop |
|---|---|
| Prompt | image |
| Small | image |

Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

File: src/components/Modal/Modal.vue [hunk 1]
```diff
@@ -295,7 +295,7 @@ export default {
 			type: String,
 			default: 'normal',
 			validator: size => {
-				return ['small', 'medium', 'normal', 'large', 'full'].indexOf(size) !== -1
+				return ['prompt', 'small', 'medium', 'normal', 'large', 'full'].indexOf(size) !== -1
```
Comment (Pytal) [inline src/components/Modal/Modal.vue:298]:
```suggestion
				return ['prompt', 'small', 'medium', 'normal', 'large', 'full'].includes(size)
```
Clearer semantics :)

Comment (szaimen) [reply src/components/Modal/Modal.vue]: fixed

**Comment by szaimen** [comment]
Now, the collection of sizes looks like this:

| Size | Desktop |
|---|---|
| Prompt | image |
| Small | image |
| Medium | image |
| Normal | image |
| Large | image |
| Full | image |

**Review by Pytal — APPROVED** [review]

**Review by marcoambrosini — APPROVED** [review]
Very nice!

**Pull request merged**